src/configSchema.ts

```diff
@@ -184,4 +203,10 @@ export const VALIDATION_SCHEMA = z
 	.refine(
 		(config) => config.matchMode === MatchMode.SAFE || config.linkDir,
 		ZodErrorMessages.needsLinkDir,
-	);
+	)
+	.refine((config) => {
+		if (config.linkDir && config.dataDirs) {
```

Comment:
Will it handle an empty array for dataDirs?

Reply:
Yes